Bottom: f1f46a9fecbb2ddf69ede5f91adac965d372d981
Top:    e401291a2b8aa24d37ca7891d028425ddd6777fc
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-24 00:09:54 +0100

fontmode.c: Fixup warnings due to assigned but unused variables


---

diff --git a/src/find.c b/src/find.c
index c5159ba..92c8b35 100644
--- a/src/find.c
+++ b/src/find.c
@@ -1927,13 +1927,8 @@ static bool
 LookupLOConnectionsToArc (ArcTypePtr Arc, Cardinal LayerGroup)
 {
   Cardinal entry;
-  LocationType xlow, xhigh;
   struct lo_info info;
 
-  /* the maximum possible distance */
-  xlow = Arc->BoundingBox.X1 - MAX (MAX_PADSIZE, MAX_LINESIZE) / 2;
-  xhigh = Arc->BoundingBox.X2 + MAX (MAX_PADSIZE, MAX_LINESIZE) / 2;
-
   info.arc = *Arc;
   EXPAND_BOUNDS (&info.arc);
   /* loop over all layers of the group */
diff --git a/src/fontmode.c b/src/fontmode.c
index 69d61ea..2575a33 100644
--- a/src/fontmode.c
+++ b/src/fontmode.c
@@ -220,11 +220,10 @@ FontSave (int argc, char **argv, int Ux, int Uy)
       LineType *l = ii->data;
       int x1 = l->Point1.X;
       int y1 = l->Point1.Y;
-      int ox, oy, s;
+      int ox, s;
 
       s = XYtoSym (x1, y1);
       ox = (s % 16 + 1) * CELL_SIZE;
-      oy = (s / 16 + 1) * CELL_SIZE;
       symbol = &PCB->Font.Symbol[s];
 
       x1 -= ox;
